src/DataTable/TableRow.jsx

```diff
@@ -5,21 +5,32 @@ import classNames from 'classnames';
 import TableCell from './TableCell';
 
 const TableRow = ({
-  getRowProps, cells, id, isSelected,
+  getRowProps, cells, id, isSelected, isExpanded, subcomponent,
 }) => (
-  <tr {...getRowProps({
-    className: classNames({
-      'pgn__data-table-row': true,
-      'is-selected': isSelected,
-    }),
-  })}
-  >
-    {cells.map(cell => <TableCell {...cell} key={`${cell.column.Header}${id}`} />)}
-  </tr>
+  <>
+    <tr {...getRowProps({
+      className: classNames({
+        'pgn__data-table-row': true,
+        'is-selected': isSelected,
+      }),
+    })}
+    >
+      {cells.map(cell => <TableCell {...cell} key={`${cell.column.Header}${id}`} />)}
+    </tr>
+    {isExpanded && subcomponent ? (
+      <tr>
+        <td colSpan="100">
```

Comment:
Per the react-table docs, I think the `colSpan` may make more sense as `colSpan={visibleColumns.length}` so that the column is spanning the correct number of columns rather than an arbitrary large number.

`visibleColumns` should be accessible via the `react-table` instance.